spec/configuration_spec.rb

```diff
@@ -54,11 +54,60 @@
                                        short_option: "-f",
                                        description: "bar")
         ]
+
         expect do
           FastlaneCore::Configuration.create(conflicting_options, {})
         end.to raise_error "Multiple entries for short_option '-f' found!".red
       end
 
+      it "raises an error for unresolved conflict between options" do
+        conflicting_options = [
+          FastlaneCore::ConfigItem.new(key: :foo,
+                                       short_option: "-f",
+                                       conflicting_options: [:bar, :oof]),
+          FastlaneCore::ConfigItem.new(key: :bar,
+                                       short_option: "-b"),
+          FastlaneCore::ConfigItem.new(key: :oof,
+                                       short_option: "-o")
+        ]
+
+        values = {
+            foo: "",
+            bar: ""
+        }
+
+        expect do
+          FastlaneCore::Configuration.create(conflicting_options, values)
+        end.to raise_error "Unresolved conflict between options: '-f' and '-b'".red
```

Comment:
Shouldn't this print out the long name `bar` and `oof` to make it clear for the user? When passing an option via `fastlane`, the user doesn't even know the short codes.